internal/pkg/model/manifest.go

```diff
@@ -292,3 +294,11 @@ func (c *ConfigManifest) AddRelation(relation Relation) {
 func (r *ConfigRowManifest) AddRelation(relation Relation) {
 	r.Relations.Add(relation)
 }
+
+func (r *ConfigManifest) GetMetadata() *orderedmap.OrderedMap {
+	return r.Metadata
+}
+
+func (c *ConfigManifest) SetMetadata(metadata *orderedmap.OrderedMap) {
```

Comment:
Tato metoda je tiez zbytocna.

Reply:
Ok, upravím (ale opisoval jsem teda tady z relací 😃 https://github.com/keboola/keboola-as-code/blob/4dae34027234865679b06232162a55a77bcce5a4/internal/pkg/model/manifest.go#L274-L284)

Reply:
Tam je to pre `ObjectManifestWithRelations` interface. Co z kodu nevidno, ale v IDE ano.